strings/power_for_large_numbers_represented_as_strings.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * \file
+ * \brief The gfg
+ *modulo-power-for-large-numbers-represented-as-strings
+ */
+#include <iostream> 
+using namespace std; 
+#define ll long long int 
+const ll MOD = 1e9 + 7; 
+// Returns modulo exponentiation for two numbers 
+// represented as long long int. It is used by 
+// powerStrings(). Its complexity is log(n) 
+ll powerLL(ll x, ll n) { 
+	ll result = 1; 
+	while (n)
+	 { 
+		if (n & 1) 
+			result = result * x % MOD; 

```

Comment:
Please make it look like C++ code, not C
```suggestion
		if (n & 1) {
			result = result * x % MOD;
		}

```
Same in other parts of the code.